doc/code/WatchFaces.md

```diff
@@ -0,0 +1,97 @@
+# Add Watchfaces 
+
+This page will help you to add a new Watch Face to InfiniTime.
+
+## Theory
+
+This example uses the existing Digital Watch Face, and copies it to a new watch face as a starting point.
+
+## Files 
+
+- displayapp/UserApps.h
+- displayapp/apps/Apps.h.in
+- displayapp/apps/CMakeLists.txt
+- displayapp/screens/WatchFaceNew.cpp 
+- displayapp/screens/WatchFaceNew.h 
+- CMakeLists.txt
+
+### Branch and Copy 
+
+Use git to fork and get a local copy (commands not shown). 
+
+Then with all the submodules updated add a branch and copy the watch face files
+
+```bash
+git submodule update --init
+git switch -c Watchface_New
+cp ./src/displayapp/screens/WatchFaceDigital.cpp ./src/displayapp/screens/WatchFaceNew.cpp
+cp ./src/displayapp/screens/WatchFaceDigital.h ./src/displayapp/screens/WatchFaceNew.h
+```
+
+### Amend Files
+
+Use sed to update `WatchFaceDigital` in the newly copied files with `WatchFaceNew` as well as update the `WatchFace::Digital` type to `WatchFace::NewType`.
```

Comment:
No need to be imply this requires sed. You can tell the reader to copy the files and replace the text and then show the sed as a suggestion. Some people will want to do that in their editor or honestly not know the Unix tooling. (yeah, I know, but not everyone loves the terminal like us ....)